### PR TITLE
Create installable package

### PR DESCRIPTION
**Pull request by kabilar**
## Description

This pull request mirrors the changes in the [datajoint/intanrhsreader](https://github.com/datajoint/intanrhsreader) repository to make this repository ([datajoint/intanrhdreader](https://github.com/datajoint/intanrhdreader)) an installable package that can read Intan RHD files.

## Changes

See `changelog.md` for a comprehensive list of all changes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+from os import path
+from setuptools import setup, find_packages
+
+
+pkg_name = "intanrhdreader"
+here = path.abspath(path.dirname(__file__))
```
Comment (dimitri-yatsenko) [inline setup.py:7]: are we standardizing around `pathlib` or `os.path`?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+with open(path.join(here, "README.md"), "r") as f:
+    long_description = f.read()
+
+with open(path.join(here, "requirements.txt")) as f:
+    requirements = f.read().splitlines()
+
+with open(path.join(here, "version.py")) as f:
```
Comment (dimitri-yatsenko) [inline setup.py:15]: why not import `version.py`?

Comment (kabilar) [reply setup.py:15]: I am not sure.  DataJoint Python and the Elements are all configured this way.

File: setup.py (hunk 1 above)
Comment (kabilar) [reply setup.py:7]: We haven't.  Both DataJoint Python and Elements use both.  Do you have preference of which package to use?

**Comment by ttngu207** [comment]
Merge to move forward - minor comments to be addressed in next PRs

**Pull request merged**